src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,107 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+interface Props {
+  items: string[];
+  ItemsPerPage: number;
+  currentPage: number;
+  setCurrentPage: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  ItemsPerPage,
```

Comment:
The prop `ItemsPerPage` should be renamed to `itemsPerPage` to follow the camelCase naming convention commonly used in JavaScript and TypeScript for variable and prop names.